database/factories/UserFactory.php

```diff
@@ -19,11 +19,19 @@
 $factory->define(Account::class, function (Faker $faker) {
     return [
         'name' => $faker->name,
+        'gender'=> '1',
+        'birthday' => $faker->dateTimeAD,
+        'birthplace' => $faker->address,
+        'resident' => $faker->address,
         'email' => $faker->unique()->safeEmail,
         'email_verified_at' => now(),
-        'password' => bcrypt('12345678'), // password
+        'password' => '12345678', // password
```

Comment:
password dùng bcrypt chứ ko lưu thẳng vào db nhé.

Reply:
Vì ở bên model bọn e có xử lý 
public function setPasswordAttribute($password)
    {
        $this->attributes['password'] = bcrypt($password);
    }
nên trong này e làm vậy luôn :)

Reply:
@donhatptit em có chắc là để cái setPasswordAttribute thì khi chạy seed nó sẽ vào đấy ko? hay nó ko vào?

Reply:
Có ạ. e thử rồi ạ :v